packages/wallet-management/src/connectors/safeWalletConnector.ts

```diff
@@ -0,0 +1,88 @@
+import { ethers } from 'ethers';
+import events from 'events';
+import type {
+  AccountData,
+  InjectedConnectorConstructorArgs,
+  Wallet,
+} from '../types';
+
+import type { SafeInfo } from '@safe-global/safe-apps-sdk';
+import { SafeAppProvider } from '@safe-global/safe-apps-provider';
+import SafeAppsSDK from '@safe-global/safe-apps-sdk/dist/src/sdk';
+
+export class SafeWalletConnector
+  extends events.EventEmitter
+  implements Partial<Wallet>
+{
+  public provider: ethers.providers.Web3Provider | undefined;
+
+  public isActivationInProgress: boolean = false;
+  public account: AccountData | undefined;
+
+  public name: string;
+  public icon: string;
+
+  constructor(args: InjectedConnectorConstructorArgs) {
+    super();
+    this.name = args.name;
+    this.icon = args.icon;
+
+    this.calcAccountData();
+  }
+
+  public installed = () => {
```

Comment:
added these placeholder methods to keep the interface of the `Wallet` consistent

Reply:
can we add some realy logic here? to detect if its running in the safe app iframe? cause we really do use that function. and if this just returns false, we won't show the wallet